model/src/test/kotlin/licenses/TestData.kt

```diff
@@ -43,6 +43,7 @@ import org.ossreviewtoolkit.utils.spdx.toSpdx
 import org.ossreviewtoolkit.utils.test.scannerRunOf
 
 val authors = setOf("The Author", "The Other Author")
+val authorsWithCopyright = setOf("Copyright (C) 2024 The Author", "The Other Author")
```

Comment:
Why does the variable name say "...WithCopyright" if at least the second entry comes without a Copyright?